website/docs/partials/_jwt-generation-partial.md

```diff
@@ -15,24 +16,30 @@ The HTTP connection between your beacon node and execution node needs to be auth
   <TabItem className="unclickable-element" value="label"></TabItem>
   <TabItem value="win">
 
-```
-## Optional. This command is necessary only if you've previously configured USE_PRYSM_VERSION
-SET USE_PRYSM_VERSION=v4.0.0
-
-## Required.
-prysm.bat beacon-chain generate-auth-secret
-```
+  ## Optional. This command is necessary only if you've previously configured USE_PRYSM_VERSION
+  <code>
+  SET USE_PRYSM_VERSION=<PrysmVersion/>
+  </code>
+ 
+  ## Required.
+  <code>
+  prysm.bat beacon-chain generate-auth-secret
```

Comment:
Also on Windows the display looks weird:
![image](https://github.com/prysmaticlabs/documentation/assets/4943830/82cb0f51-fc5a-4a10-bfe7-be08d1b0f60d)